api/terms/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" prefix="skos: http://www.w3.org/2004/02/skos/core#">
+  <head>
+    <meta charset="UTF-8">
+    <title>iRail terminology</title>
+  </head>
+  <body typeof="skos:ConceptScheme" resource="https://api.irail.be/terms/">
+    <h1 property="dct:title">iRail terminology</h1>
+    <p>Terminology used throughout the iRail project can be found here:</p>
+    <section id="occupancy" typeof="skos:Concept">
+      <h2 resource="skos:prefLabel">Occupancy</h2>
+      <ul>
+        <li typeof="skos:inScheme" resource="http://api.irail.be/terms/unknown"><a href="http://api.irail.be/terms/unknown">terms/unknown</a>. <span typeof="skos:prefLabel">Unknown: there is insufficient information to define an occupancy level.</span></li>
```

Comment:
typeof should be `rel` here. Rel will make sure a triple is generated as follows:
```turtle
<resource> skos:inScheme <subjectofthehigherblock>.
# Or in this case:
<http://api.irail.be/terms/unknown> skos:inScheme <http://api.irail.be/terms> .
```